parser.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python3
+# parser.py
+
+
+def parse(text):
+    lines = []
+    for line in text:
+        if len(line)>0:
```

Comment:
Minor nitpick: `len(line) > 0`

Reply:
I think `if line` works and is a bit more Pythonic ;)